demo-certs/create_certs.sh

```diff
@@ -90,6 +98,14 @@ echo "creating JKS trust store ($DIR/$HONO_TRUST_STORE) containing CA certificat
 keytool -import -trustcacerts -noprompt -alias root -file $DIR/root-cert.pem -keystore $DIR/$HONO_TRUST_STORE -storepass $HONO_TRUST_STORE_PWD
 keytool -import -trustcacerts -noprompt -alias ca -file $DIR/ca-cert.pem -keystore $DIR/$HONO_TRUST_STORE -storepass $HONO_TRUST_STORE_PWD
 
+echo ""
+echo "updating example-tenants.json with CA public key "
```

Comment:
I would rather not introduce this dependency on the deploy module here
Instead, we might be able to add the trust anchor as part of the initial creation of the `device-registry` volume when starting up/deploying the device registry?

Reply:
> Instead, we might be able to add the trust anchor as part of the initial creation of the `device-registry` volume when starting up/deploying the device registry?

This proved to be rather difficult, because it requires to install tools (jq and openssl) into the init container which is a pain with the busybox image. Instead I decoupled the modules by writing the CA's public key into a file and then use it to update example-tenants.json in the build process via resource filtering.